README.md

```diff
@@ -45,8 +45,31 @@ For other systems, please check the above requirements and use your correspondin
 TBD
 
 ## Windows
-
-TBD
+### GL-Node Builder
+- get [glbsp executable](http://glbsp.sourceforge.net/download.php)
+- place it in the root of the goom directory
+
+### mingw-w64 for makefile execution and portmidi usage
+- install [mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds)
+- add *\<mingw64 dir\>/mingw64/bin* to your user- or system path
+- you now can use *mingw32-make*
+
+### Sound (TODO: this surely can be done more straight forward)
+- download and install [cmake](https://cmake.org/download/) (or use chocolatey `choco install cmake`)
+- get [the sources for portmidi](https://sourceforge.net/projects/portmedia/files/portmidi/) and unzip them
+- use the cmake gui (in *\<cmake dir\>\\bin*) to configure and generate build files
```

Comment:
hmm.. maybe we should really think about switching to SDL.

(gitter integration test)